src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -124,6 +125,9 @@ public function loadAdminScripts()
                         'Are you sure you want to disconnect your PayPal account?',
                         'give'
                     ),
+                    'keepWebhooksAfterDisconnect' => esc_html__('Continue to receive events, including subscription renewals, from old account?',
+                        'give'),
+                    'keepWebhooksAfterDisconnectLearnMore' => esc_html__('Learn More', 'give') . ' &raquo;',
```

Comment:
```suggestion
                    'keepWebhooksAfterDisconnectLearnMore' => esc_html__('Learn why this is critically important.', 'give') . ' &raquo;',
```

Reply:
Resolved in 6a731f5af2093f2d592e242e51c7c12cd4079483